components/Heart.tsx

```diff
@@ -16,27 +16,29 @@ export default function Heart(props: {
 
   if (props.isLoading) {
     return (
-      <div>
-        <button className={styles.heart} onClick={handleClick}>
-          <Icon data={favorite_filled} />
-        </button>
+      <div className={styles.heart}>
+        <Icon data={favorite_filled} />
       </div>
```

Comment:
shouldn't be actionable while loading... so turning it into a div instead of a button.